assignments/lambda-classes.js

```diff
@@ -1 +1,133 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(options) {
+    this.name = options.name;
+    this.age = options.age;
+    this.location = options.location;
+    this.gender = options.gender;
+    }
+    speak() {
+        return `Hello my name is ${this.name}, I am from ${this.location}`;
+    }
+}//Person class
+
+class Instructor extends Person{
+    constructor(InstructorOptions) {
+    super(InstructorOptions);
```

Comment:
The innards of the constructor don't need a special name. Usually it will be a generic `props` for every constructor. It's a useful exercise to know that those options are being used to create an instructor, but you'll come across `props` more often than anything detailed.